inference-engine/thirdparty/clDNN/runtime/ocl/ocl_device.cpp

```diff
@@ -223,12 +223,6 @@ device_info init_device_info(const cl::Device& device) {
     info.supports_subgroups_short = extensions.find("cl_intel_subgroups_short") != std::string::npos;
     info.supports_subgroups_char = extensions.find("cl_intel_subgroups_char") != std::string::npos;
 
-    info.supports_imad = get_imad_support(device);
-    info.supports_immad = false;
```

Comment:
You should initialize values first, otherwise the values will be undefined, which may lead to incorrect results here:
```
        info.supports_imad = info.supports_imad || (features & CL_DEVICE_FEATURE_FLAG_DP4A_INTEL);
        info.supports_immad = info.supports_immad || (features & CL_DEVICE_FEATURE_FLAG_DPAS_INTEL);
```

Reply:
Thanks, fixed.